Vocable/Extensions/UIColor+AppExtensions.swift

```diff
@@ -11,22 +11,28 @@ import UIKit
 extension UIColor {
     
     // MARK: New Branded Colors
-    
-    static let primaryColor = UIColor(named: "Primary")!
-    
-    static let defaultTextColor = UIColor(named: "DefaultFontColor")!
+
+    private convenience init?(safelyNamed name: String) {
```

Comment:
Can we document the workaround with something like 

```suggestion
    // Workaround for https://openradar.appspot.com/47113341. Prevent crashing the IBDesignables agent when using a color from the asset catalog.
    private convenience init?(safelyNamed name: String) {
```

Reply:
This was a great idea you had. Thank you for the suggestion.

Reply:
Always happy to help 😉